anomaly-detection/build.gradle

```diff
@@ -0,0 +1,109 @@
+
+plugins {
+    id 'com.github.johnrengelman.shadow' version '1.2.4'
+}
+
+apply plugin: "java"
+apply plugin: "distribution"
+apply plugin: 'application'
+
+sourceCompatibility = "1.8"
+mainClassName = 'io.pravega.anomalydetection.ApplicationMain'
+applicationDefaultJvmArgs = ["-Dlogging.config=file:conf/logback.xml"]
+
+archivesBaseName = 'pravega-flink-anomaly-detection'
+
+dependencies {
+    compileOnly "org.slf4j:slf4j-api:1.7.14"
+    compileOnly "ch.qos.logback:logback-classic:1.1.7"
+    compile "io.pravega:pravega-connectors-flink_2.11:${pravegaVersion}"
+    compile "org.apache.flink:flink-streaming-java_2.11:${flinkVersion}"
+    compile "org.apache.flink:flink-connector-elasticsearch5_2.11:${flinkVersion}"
+}
+
+shadowJar {
+    //configurations = [project.configurations.runtime]
+    dependencies {
+        include dependency("io.pravega:pravega-connectors-flink_2.11")
+
+        //All below dependencies are from Elastic Search 5 connector
```

Comment:
Did this list come from somewhere?  Can you link to it, so we know how to update this over time?